docs/tutorials/initializedirtyimage.py

```diff
@@ -0,0 +1,190 @@
+from astropy.utils.data import download_file
+from torch.utils.tensorboard import SummaryWriter
+
+# In this tutorial we will be saving and loading the model we use and it is important to note that when saving and loading a model the cell_size, nchan, and npix are the same between the model and the coordinates if used in another document.
+
+# +
+# load the mock dataset of the ALMA logo
+fname = download_file(
+    "https://zenodo.org/record/4498439/files/logo_cube.npz",
+    cache=True,
+    show_progress=True,
+    pkgname="mpol",
+)
+
+# this is a multi-channel dataset... but for demonstration purposes we'll use
+# only the central, single channel
+chan = 4
+d = np.load(fname)
+uu = d["uu"][chan]
+vv = d["vv"][chan]
+weight = d["weight"][chan]
+data_re = d["data_re"][chan]
+data_im = -d["data_im"][
+    chan
+]  # we're converting from CASA convention to regular TMS convention by complex conjugating the visibilities
+
+# define the image dimensions, making sure they are big enough to fit all
+# of the expected emission
+coords = coordinates.GridCoords(
+    cell_size=0.03, npix=180
+)  # The smaller the cell size and the larger the npix can greatly increase run time
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data_re,
+    data_im=data_im,
+)
+
+# export to PyTorch dataset
+dset = gridder.to_pytorch_dataset()
+# -
+
+# Show the dirty image
+img, beam = gridder.get_dirty_image(weighting="briggs", robust=0.0, unit="Jy/beam")
+kw = {"origin": "lower", "extent": gridder.coords.img_ext}
+fig, ax = plt.subplots(ncols=1)
+snp = ax.imshow(np.squeeze(img), **kw)
+ax.set_title("image")
+ax.set_xlabel(r"$\Delta \alpha \cos \delta$ [${}^{\prime\prime}$]")
+ax.set_ylabel(r"$\Delta \delta$ [${}^{\prime\prime}$]")
+plt.colorbar(snp)
+
+# ### Model and Optimization Setup
+#
+# Here we will be setting up the optimizer and the image model (RML). If this is unfamiliar please reference the Optimization tutorial. The initial parameters of the model are also displayed and can be contrasted with them after the optimization loop. }
+
+dirty_image = torch.tensor(img.copy())  # turns it into a pytorch tensor
+rml = precomposed.SimpleNet(coords=coords, nchan=dset.nchan)
+optimizer = torch.optim.SGD(
+    rml.parameters(), lr=500.0
+)  # multiple different possiple optimizers
+rml.state_dict()  # parameters of the model
+
+
+# ### Loss Function
+
+# The loss function that will be used to optimize the initial part of the image is the pixel-to-pixel L2 norm (also known as the Euclidian Norm). It calculates the loss based off of the image-plane distance between the dirty image and the state of the ImageCube in order to make the state of the ImageCube closer to the dirty image.
+#
+# The L2 Norm (Euclidian Norm) is based off of the euclidian distance function and it takes the form of: $ L_{oss} = \sqrt{\sum_{i=1}^{N} (y_i - x_i)^2}  $
+
+
+def l2norm(sky, dirt):
```

Comment:
This loss function works, but I think it would be simpler to just use `loss_fn = torch.nn.MSELoss()`